card/mod/standard/set/abstract/search.rb

```diff
@@ -92,14 +92,58 @@ def implicit_item_view
 end
 
 format :csv do
-  view :card_list do |args|
-    items = super args
+  view :nested_fields do
```

Comment:
@ethn Any thoughts on the view name? Maybe just "fields" or "export_items", "item_fields"?